src/env.ts

```diff
@@ -37,26 +38,31 @@ for (const k in defaults) {
   }
 }
 
-class configuration {
+export default class configuration {
+  TRIALSCOPE_ENDPOINT: string;
+  PORT: number;
+  TRIALSCOPE_TOKEN: string | null;
   constructor() {
     // Default to defaults
-    for (const k in defaults) {
-      this[k] = defaults[k];
-    }
+    this.TRIALSCOPE_ENDPOINT = defaults.TRIALSCOPE_ENDPOINT;
+    this.TRIALSCOPE_TOKEN = defaults.TRIALSCOPE_TOKEN;
+    this.PORT = parseInt(defaults.port);
```

Comment:
This doesn't appear to set the port number when running the server. It errors out claiming there isn't a port set.